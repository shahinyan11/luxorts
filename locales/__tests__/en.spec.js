import en from '../en';

it('en locale matches snapshot', () => {
  expect(en).toMatchSnapshot();
});
