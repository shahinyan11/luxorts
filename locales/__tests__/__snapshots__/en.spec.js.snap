// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`en locale matches snapshot 1`] = `
Object {
  "locale": "en",
  "messages": Object {
    "aboutUs.joinUsOnOurSocialNetworks": "Join us on our social networks.",
    "aboutUs.letsGetSocial": "Let's get social",
    "aboutUs.ourMission": "Our mission",
    "aboutUs.thePowerOfOurPlatform": "The power of our platform",
    "aboutUs.title": "Get to know {styledWord}",
    "aboutUs.toPowerGlobalTravel": "To power global travel for everyone, everywhere.",
    "aboutUs.weAreTravelersAndTechnologists": "We are travelers and technologists. We're used to breaking things down and building them back up again until they're even better. We know travel can be hard, but we also know that it's worth it every time. And because we believe travel is a force for good, we take our roles seriously. We're here to build a great product, and facilitate connections between travelers and our partners that truly bring good into the world.",
    "aboutUs.weHelpOurTravelers": "We help our travelers to find the right pathways through millions of possibilities to reach the best possible outcome.",
    "aboutUs.whenWePowerMoreTravel": "When we power more travel, we unleash more opportunities to strengthen connections, broaden horizons and bridge divides.",
    "accountSettings.title": "Account settings",
    "addNewListing.accommodation.description": "Check that you have enough beds to accommodate all your guests comfortably.",
    "addNewListing.accommodation.title": "How many guests can your place accommodate?",
    "addNewListing.amenities.additionalAmenity.success": "Your amenity request has been saved",
    "addNewListing.amenities.title": "What amenities do you offer for guests?",
    "addNewListing.capabilities.additionalCapability.success": "Your capability request has been saved",
    "addNewListing.capabilities.title": "What capabilities do you offer for guests?",
    "addNewListing.location.title": "Where is your apartment located?",
    "addNewListing.locationMap.description": "If needed you can adjust the map so the pin is in the right location. Only confirmed guests will see this, so they will know how to get to your place.",
    "addNewListing.locationMap.title": "Is the pin in the right place?",
    "addNewListing.propertyType.place.title": "Type of place guests will have",
    "addNewListing.propertyType.title": "What kind of place are you listing?",
    "addNewListing.services.additionalService.success": "Your service request has been saved",
    "addNewListing.services.title": "What services do you offer for guests?",
    "bathroomArrangements.description": "Sharing the types of baths in each bathroom can help guests understand the bathroom arrangements.",
    "bathroomArrangements.title": "Bathroom arrangements",
    "contactUs.haveAnyQuestions": "Have any questions? We'd like to hear from you.",
    "contactUs.letsStartConversation": "Let's start a conversation",
    "contactUs.message.placeholder": "How we can help?",
    "contactUs.title": "Get in touch",
    "edit.listing.availability.description": "Manage availability and pricing for your place.",
    "emailConfirmation.enterCode": "Enter security code from the email to confirm your email address.",
    "emailConfirmation.title": "Confirm your email",
    "faq.browseAllTopics": "Browse all topics",
    "faq.description": "Browse help topics to find answers to your questions.",
    "faq.findCategorySpecificContent": "Find category-specific content in these other sections of the FAQ.",
    "faq.notFindingWhatYouNeed": "Not finding what you need?",
    "faq.question.bookingTrip.text": "Welcome to Luxorts! From here, you can go anywhere - but first, a little help to get you on...",
    "faq.question.bookingTrip.title": "Booking a trip: What to do if you're new?",
    "faq.question.cancelMyReservation.text": "You can cancel a reservation any time before or during your trip.",
    "faq.question.cancelMyReservation.title": "How do I cancel my reservation for a place to stay?",
    "faq.question.creatingAccount.title": "Creating an account",
    "faq.question.creatingAnAccount.text": "Creating an Airbnb account is free and easy. Just go to luxorts.com, click Sign Up, and follow the...",
    "faq.question.payOutsideOfThePlatform.text": "If someone asks to pay for your reservation outside of (ex: a wire or bank transfer), you ne...",
    "faq.question.payOutsideOfThePlatform.title": "What should I do if someone asks me to pay outside of the platform?",
    "faq.question.updateOrRecoverPassword.text": "Forgot your password or having trouble signing in? We've been there. Time for recovery.",
    "faq.question.updateOrRecoverPassword.title": "Update or recover your password",
    "faq.question.whatIsLuxorts.text": "Luxorts helps make sharing easy, enjoyable, and safe. We verify personal profiles and listings, ...",
    "faq.question.whatIsLuxorts.title": "What is Luxorts and how does it work?",
    "faq.title": "We love to help",
    "footer.allRightsReserved": "Â© {year} Luxorts. All rights reserved.",
    "footer.clickHereToDeactivate": "Click here to deactivate",
    "footer.needToDeactivateYourAccount": "Need to deactivate your account?",
    "footer.termsAndPolicy": " By signing up, I agree to Luxorts {termsLink} and {policyLink}.",
    "home.discoverAndBookApartments": "Discover and book the perfect apartments for a vacation or business trip.",
    "home.findYour": "Find your ",
    "home.hosting": "hosting",
    "home.listYourProperty": "List your property",
    "home.listYourPropertyText": "List your property on Luxorts and open your door to rental income.",
    "home.perfectApartments": "perfect apartments",
    "home.popularApartmentsText": "Here are some of the popular apartments near you",
    "home.popularApartmentsTitle": "Popular apartments near you",
    "home.propertyTypesText": "Find spaces that suit your style.",
    "home.propertyTypesTitle": "Browse by property type",
    "home.trendingLocationsText": "The world is your playground when you're traveling. Check out a few trending locations.",
    "home.trendingLocationsTitle": "Trending locations",
    "home.tryHosting": "Try {styledWord} on Luxorts",
    "inviteFriends.info.description": "Email friends who've never tried Luxorts, and we'll send them a reminder, too.",
    "listing.availability.description": "Just select a date to block or unblock it. You can always make changes after you publish.",
    "listing.describe.description": "Catch guests' attention with a listing title and description that highlights what makes your place special.",
    "listing.describe.info": "This title will be visible only to you",
    "listing.describe.title": "Describe your place",
    "listing.houseRules.additional.success": "Your house rule request has been saved",
    "listing.houseRules.description": "Guests must agree to your house rules before they book.",
    "listing.houseRules.title": "Set house rules for your guests",
    "listing.photos.imageFormat": "Image format: JPG or PNG. Photo quality: Min. 1024 x 683, Max. 3840 x 2160.",
    "listing.pricingAndDiscounts.extraCharges.description": "Set extra charges for your guests.",
    "listing.pricingAndDiscounts.extraCharges.tooltip": "This fee will apply to every reservation",
    "listing.pricingAndDiscounts.monthlyDiscount.tooltip": "Most travelers stay longer than one month, booking listings with discounts greater than 25%",
    "listing.pricingAndDiscounts.pricePerDay.description": "Set competitive base price. You can always set different pricing for specific days in your calendar.",
    "listing.pricingAndDiscounts.weeklyDiscount.description": "Encourage guests to stay longer by offering a weekly or monthly discount - you'll spend less time turning over your space between guests.",
    "listing.pricingAndDiscounts.weeklyDiscount.tooltip": "To help increase your chances of getting weekly stays, try setting a discount",
    "listing.reservationPreferences.approvalPolicy.description": "Select your default booking approval policy.",
    "listing.reservationPreferences.approvalPolicy.title": "Booking approval policy",
    "listing.reservationPreferences.cancellationPolicy.description": "Select your policy for trip cancellations by guests.",
    "listing.reservationPreferences.cancellationPolicy.title": "Cancellation policy",
    "listing.reservationPreferences.checkIn.description": "Select a time range when your guests can check in.",
    "listing.reservationPreferences.checkIn.title": "When can guests check in?",
    "listing.reservationPreferences.inAdvance.caption": "Your entire calendar will be blocked by default, which means you'll have to manually unblock dates to get booked.",
    "listing.reservationPreferences.inAdvance.description": "Most hosts keep their calendars updated up to 6 months out.",
    "listing.reservationPreferences.inAdvance.title": "How far in advance can guests book?",
    "listing.reservationPreferences.preparationTime.description": "The availability of your listings largely depends on the length of preparation time you need between reservations.",
    "listing.reservationPreferences.preparationTime.title": "How much time do you need to prepare your space between reservations?",
    "listing.reservationPreferences.prior.description": "At least 1-day notice can help you plan for guests' arrival, but you might miss out on last-minute trips.",
    "listing.reservationPreferences.prior.title": "When do you need to be notified prior to the arrival of the guests?",
    "listing.reservationPreferences.stay.description": "Short trips can mean more bookings, but you'll turn over your space more often.",
    "listing.reservationPreferences.stay.title": "How long can guests stay?",
    "listing.reservationPreferences.title": "Set reservation preferences for your apartment",
    "listing.status.modal.publish.description": "It will become visible on the platform after you publish it.",
    "listing.status.modal.publish.subtitle": "Are you sure you want to publish this listing?",
    "listing.status.modal.publish.success": "Listing has been published",
    "listing.status.modal.publish.title": "Publish listing",
    "listing.status.modal.remove.description": "All of the leads and property information will be lost.",
    "listing.status.modal.remove.subtitle": "Are you sure you want to remove this listing?",
    "listing.status.modal.remove.success": "Listing has been removed",
    "listing.status.modal.remove.title": "Remove listing",
    "listing.status.modal.republish.description": "It will become visible on the platform after moderation is completed.",
    "listing.status.modal.republish.subtitle": "Are you sure you want to republish this listing?",
    "listing.status.modal.republish.title": "Republish listing",
    "listing.status.modal.unpublish.description": "It will not be visible on the platform after you unpublish it.",
    "listing.status.modal.unpublish.subtitle": "Are you sure you want to unpublish this listing?",
    "listing.status.modal.unpublish.success": "Listing has been unpublished",
    "listing.status.modal.unpublish.title": "Unpublish listing",
    "loginAndSecurity.description": "Enter the new unique password to protect your account.",
    "loginAndSecurity.info.description": "Settings and recommendations to help you keep your account secure.",
    "loginAndSecurity.lastUpdateTime": "Last updated {time}",
    "loginAndSecurity.title": "Login & security",
    "newPassword.description": "Please set a new password for your account.",
    "notification.avatarRemoved": "Your avatar has been removed",
    "notification.avatarUploaded": "Your avatar has been uploaded",
    "notification.deactivateAccount": "Your account has been deactivated. We have sent the notification to your email address.",
    "notification.inviteFriends": "Invite has been sent",
    "notification.personalInformationUpdated": "Personal Information has been successfully updated",
    "notification.phoneNumberUpdated": "Your phone number has been updated",
    "notification.someFriendsAreNotInvited": "Some of your friends already have accounts in the system. They will not receive an invitation",
    "notification.updatePassword": "Your password has been updated",
    "password.medium": "You can use uppercase characters, numbers and symbols to make your password stronger",
    "password.min": "Use a minimum password length of {min} or more characters",
    "password.strong": "Your password is strong enough!",
    "password.tooltip": "Use a minimum password length of {min} or more characters, including capital letters and numbers",
    "personalInformation.about.description": "Add information about yourself.",
    "personalInformation.address.description": "Use a permanent address where you can receive mail.",
    "personalInformation.dateOfBirth.description": "To use the service, you need to be at least 18. Your date of birth won't be shared with other people",
    "personalInformation.description": "Some personal details and contact info can be edited, but we may ask you to verify your identity the next time you book or create a listing.",
    "personalInformation.descriptionTitle": "What info can be edited?",
    "personalInformation.driverLicense.description": "This is the name on your driver's license.",
    "personalInformation.email.description": "Use an address you'll always have access to.",
    "personalInformation.fullName.description": "This is your profile name.",
    "personalInformation.phoneNumber.description": "Use your right phone number so users and Luxorts can get in touch.",
    "personalInformation.title": "Personal information",
    "phoneConfirmation.description": "We sent a sms to {phone}",
    "phoneConfirmation.enterCode": "Enter security code from the sms to confirm your phone.",
    "phoneConfirmation.title": "Confirm your phone",
    "publish.description": "You are able to welcome your first guests starting January 6, 2022. If you would like to update your calendar or house rules, you can easily do all that after you hit publish.",
    "publish.modal.draft.description": "This listing has been saved as a draft. You can edit and publish it later.",
    "publish.modal.draft.title": "Your listing has been saved as a draft",
    "publish.modal.reviewed.description": "This listing will automatically be published on Luxorts after moderation is completed.",
    "publish.modal.reviewed.title": "Congrats! Your listing is now being reviewed",
    "resetPassword.description": "Please enter the email address you used when creating your account and we will send you a password recovery email.",
    "shared.24HourReview": "24 Hour Review",
    "shared.aYearAgo": "a year ago",
    "shared.about": "About",
    "shared.aboutUs": "About Us",
    "shared.accommodation": "Accommodation",
    "shared.accommodationCount": "Accommodation {count}",
    "shared.accountSettings": "Account settings",
    "shared.add": "Add",
    "shared.addAnotherBath": "Add another bath",
    "shared.addAnotherBed": "Add another bed",
    "shared.addBaths": "Add baths",
    "shared.addBeds": "Add beds",
    "shared.addCaption": "Add a caption",
    "shared.addGuests": "Add guests",
    "shared.addNewListing": "Add new listing",
    "shared.additionalAmenities": "Additional amenities",
    "shared.additionalCapabilities": "Additional capabilities",
    "shared.additionalRules": "Additional rules",
    "shared.additionalServices": "Additional services",
    "shared.address": "Address",
    "shared.allows24Hours": "Allows 24 hours to accept a booking request.",
    "shared.allowsBookingImmediately": "Allows booking immediately without sending a request for approval.",
    "shared.alreadyHaveAnAccount": "Already have an account?",
    "shared.amenities": "Amenities",
    "shared.amenitiesCount": "Amenities {count}",
    "shared.anyTime": "Any time",
    "shared.apply": "Apply",
    "shared.areYouSureYouWantToRemoveProfileImage": "Are you sure you want to remove profile image?",
    "shared.availability": "Availability",
    "shared.availabilityAndPricing": "Availability & pricing",
    "shared.availabilitySettings": "Availability settings",
    "shared.available": "Available",
    "shared.back": "Back",
    "shared.backToSignIn": "Back to Sign In",
    "shared.basePrice": "Base price",
    "shared.bathAmount": "{amount} bath",
    "shared.bathroomNumber": "Bathroom {number}",
    "shared.bathrooms": "Bathrooms",
    "shared.baths": "Baths",
    "shared.bathsAmount": "{amount} baths",
    "shared.bathtub": "Bathtub",
    "shared.bed": "Bed",
    "shared.bedAmount": "{amount} bed",
    "shared.bedroomAmount": "{amount} bedroom",
    "shared.bedroomNumber": "Bedroom {number}",
    "shared.bedrooms": "Bedrooms",
    "shared.bedroomsAmount": "{amount} bedrooms",
    "shared.beds": "Beds",
    "shared.bedsAmount": "{amount} beds",
    "shared.blockNumberNightBeforeAndAfterEachReservation": "Block {number} night before and after each reservation",
    "shared.blockThisMonth": "Block this month",
    "shared.blocked": "Blocked",
    "shared.booked": "Booked",
    "shared.bookings": "Bookings",
    "shared.calendar": "Calendar",
    "shared.cancel": "Cancel",
    "shared.capabilities": "Capabilities",
    "shared.change": "Change",
    "shared.changeImage": "Change Image",
    "shared.changesHaveBeenSaved": "Changes have been saved",
    "shared.checkIn": "Check In",
    "shared.checkOut": "Check Out",
    "shared.cities": "Cities",
    "shared.city": "City",
    "shared.cleaningFee": "Cleaning Fee",
    "shared.clear": "Clear",
    "shared.clearAllFilters": "Clear all filters",
    "shared.columnsSettings": "Columns settings",
    "shared.company": "Company",
    "shared.confirm": "Confirm",
    "shared.contactUs": "Contact Us",
    "shared.country": "Country",
    "shared.currentPassword": "Current Password",
    "shared.dashboard": "Dashboard",
    "shared.dateCreated": "Date Created",
    "shared.dateFormat": "DD/MM/YYYY",
    "shared.dateIn": "Date in",
    "shared.dateOfBirth": "Date Of Birth",
    "shared.dateOut": "Date out",
    "shared.datesUnavailableByDefault": "Dates Unavailable By Default",
    "shared.dayNumber": "{number} day",
    "shared.dayNumberPlural": "{number} days",
    "shared.deactivate": "Deactivate",
    "shared.deactivateAccount": "Deactivate account",
    "shared.deactivateDescription": "Enter your password to confirm you want to deactivate your account.",
    "shared.deactivateQuestion": "Are you sure you want to deactivate your account? You can restore your account anytime.",
    "shared.describeAmenity": "Describe amenity",
    "shared.describeCapability": "Describe capability",
    "shared.describeService": "Describe service",
    "shared.description": "Description",
    "shared.didNotGetIt": "Didn't get it?",
    "shared.discard": "Discard",
    "shared.done": "Done",
    "shared.dontHaveAnAccountYet": "Don't have an account yet?",
    "shared.double": "Double",
    "shared.doubleVanity": "Double vanity",
    "shared.draft": "Draft",
    "shared.dragYourItemsIn": "Drag your items in",
    "shared.edit": "Edit",
    "shared.editBaths": "Edit baths",
    "shared.editBeds": "Edit beds",
    "shared.email": "Email",
    "shared.emailHasBeenResent": "Email has been resent",
    "shared.enterAmenityTitle": "Enter amenity title",
    "shared.enterBathTitle": "Enter bath title",
    "shared.enterBedTitle": "Enter bed title",
    "shared.enterCapabilityTitle": "Enter capability title",
    "shared.enterEmail": "Enter email",
    "shared.enterRule": "Enter rule",
    "shared.enterSecurityCode": "Enter security code",
    "shared.enterSecurityCodeToSetNewPassword": "Enter security code from the email to set a new password.",
    "shared.enterServiceTitle": "Enter service title",
    "shared.entirePlace": "Entire place",
    "shared.extraCharges": "Extra charges",
    "shared.faq": "FAQ",
    "shared.favorites": "Favorites",
    "shared.female": "Female",
    "shared.finishYourListing": "Finish your listing",
    "shared.firstName": "First Name",
    "shared.flexible": "Flexible",
    "shared.forGuests": "For Guests",
    "shared.forHosts": "For Hosts",
    "shared.forgotPassword": "Forgot password?",
    "shared.free": "Free",
    "shared.from": "From",
    "shared.fullName": "Full Name",
    "shared.fullRefundNumberDay": "Full refund {number} day prior to arrival.",
    "shared.gender": "Gender",
    "shared.general": "General",
    "shared.goToListings": "Go to Listings",
    "shared.goToProfile": "Go to profile",
    "shared.guestAmount": "{amount} guest",
    "shared.guests": "Guests",
    "shared.guestsAmount": "{amount} guests",
    "shared.guestsCanBookBefore": "Guests can book before",
    "shared.help": "Help",
    "shared.hiddenTitle": "Hidden Title",
    "shared.hosts": "Hosts",
    "shared.houseRules": "House rules",
    "shared.howManyBathroomsCanYourGuestsUse": "How many bathrooms can your guests use?",
    "shared.howManyBedroomsCanYourGuestsUse": "How many bedrooms can your guests use?",
    "shared.howManyBedsCanYourGuestsUse": "How many beds can your guests use?",
    "shared.instantBook": "Instant Book",
    "shared.inviteFriends": "Invite friends",
    "shared.inviteFriendsToJoinLuxortsCommunity": "Invite friends to join Luxorts community.",
    "shared.invited": "INVITED ({count}):",
    "shared.jettedTub": "Jetted tub",
    "shared.lastName": "Last Name",
    "shared.lastUpdated": "Last Updated",
    "shared.leavePage": "Leave the page",
    "shared.leavePageConfirmation": "You are leaving this page. Would you like to save changes?",
    "shared.lengthOfStayDiscounts": "Length-of-stay discounts",
    "shared.listYourProperty": "List your property",
    "shared.listing": "Listing",
    "shared.listingPreview": "Listing preview",
    "shared.listingTitle": "Listing Title",
    "shared.listings": "Listings",
    "shared.loadingDots": "Loading...",
    "shared.location": "Location",
    "shared.locationMap": "Location map",
    "shared.logOut": "Log out",
    "shared.loginSecurity": "Login & security",
    "shared.lostYourPassword": "Lost your password?",
    "shared.luxorts": "Luxorts",
    "shared.mainPhoto": "Main photo",
    "shared.male": "Male",
    "shared.messages": "Messages",
    "shared.moderate": "Moderate",
    "shared.monthlyDiscount": "Monthly Discount",
    "shared.monthlyPriceTooltip": "Monthly price with {percents} discount: {discount}",
    "shared.n": "N",
    "shared.nameOnDriverLicense": "Name On Driver's License",
    "shared.newPassword": "New password",
    "shared.next": "Next",
    "shared.nightsMax": "Nights Max",
    "shared.nightsMin": "Nights Min",
    "shared.noListingsYet": "No listings yet.",
    "shared.noResultsFound": "No results found.",
    "shared.none": "None",
    "shared.notProvided": "Not provided",
    "shared.notifications": "Notifications",
    "shared.numberMonthsInAdvance": "{number} Months In Advance",
    "shared.numberOfGuestsYouWillAccommodate": "Number of guests you will accommodate?",
    "shared.ok": "Ok",
    "shared.onModeration": "On Moderation",
    "shared.or": "Or",
    "shared.other": "Other",
    "shared.paid": "Paid",
    "shared.password": "Password",
    "shared.paymentsPayouts": "Payments & payouts",
    "shared.percentsRefundUpUntil": "{percents} refund up until 1 week prior to arrival.",
    "shared.phoneNumber": "Phone Number",
    "shared.photos": "Photos",
    "shared.pleaseUploadYourProfileImageToPublishTheListing": "Please upload your profile image to publish the listing",
    "shared.preparationTime": "Preparation Time",
    "shared.preview": "Preview",
    "shared.priceNight": "{price} / night",
    "shared.priceYourSpace": "Price your space",
    "shared.pricingAndDiscounts": "Pricing & discounts",
    "shared.pricingSettings": "Pricing settings",
    "shared.privacyPolicy": "Privacy Policy",
    "shared.privateRoom": "Private room",
    "shared.properties": "Properties",
    "shared.propertyType": "Property type",
    "shared.providePersonalDetailsAndHowWeCanReachYou": "Provide personal details and how we can reach you.",
    "shared.publish": "Publish",
    "shared.publishListing": "Publish listing",
    "shared.published": "Published",
    "shared.queen": "Queen",
    "shared.readMore": "Read more",
    "shared.recover": "Recover",
    "shared.recoverYourPassword": "Recover your password",
    "shared.recoveryLinkHasBeenResent": "Recovery link has been resent",
    "shared.remove": "Remove",
    "shared.removeProfileImage": "Remove profile image",
    "shared.removed": "Removed",
    "shared.reports": "Reports",
    "shared.republish": "Republish",
    "shared.resendEmail": "Resend email",
    "shared.resendInvite": "Resend invite",
    "shared.reservationPreferences": "Reservation preferences",
    "shared.reviewPaymentsPayouts": "Review payments, payouts, coupons, gift cards, and taxes.",
    "shared.reviews": "Reviews",
    "shared.sameDay": "Same day",
    "shared.save": "Save",
    "shared.saveAndExit": "Save and exit",
    "shared.saveAsDraft": "Save as draft",
    "shared.searchAmenity": "Search amenity",
    "shared.searchListing": "Search listing",
    "shared.selectCountry": "Select country",
    "shared.selectGender": "Select gender",
    "shared.selectPropertyType": "Select property type",
    "shared.selectStateRegion": "Select state/region",
    "shared.selectTime": "Select time",
    "shared.sendInvite": "Send invite",
    "shared.services": "Services",
    "shared.setAvailability": "Set availability",
    "shared.setAvailabilityAndPricing": "Set availability & pricing",
    "shared.setPriceForYourSpace": "Set price for your space",
    "shared.sharedRoom": "Shared room",
    "shared.shower": "Shower",
    "shared.signIn": "Sign In",
    "shared.signUp": "Sign Up",
    "shared.single": "Single",
    "shared.sofaBed": "Sofa bed",
    "shared.stateRegion": "State/Region",
    "shared.status": "Status",
    "shared.statusCount": "Status {count}",
    "shared.street": "Street",
    "shared.strict": "Strict",
    "shared.submit": "Submit",
    "shared.suiteApartmentNumberOptional": "Suite, Apartment Number (Optional)",
    "shared.support": "Support",
    "shared.switchToHosting": "Switch to hosting",
    "shared.tableOfContents": "Table of Contents",
    "shared.termsOfService": "Terms of Service",
    "shared.timeAm": "{time}:00 AM",
    "shared.timePm": "{time}:00 PM",
    "shared.to": "To",
    "shared.today": "Today",
    "shared.tryAgain": "Try again",
    "shared.unblockThisMonth": "Unblock this month",
    "shared.unpublish": "Unpublish",
    "shared.unpublished": "Unpublished",
    "shared.update": "Update",
    "shared.updateYourPasswordAndSecureYourAccount": "Update your password and secure your account.",
    "shared.uploadImageOrDropItHere": "Upload image or drop it here",
    "shared.uploadListingPhotos": "Upload listing photos",
    "shared.uploadPhotos": "Upload photos",
    "shared.uploadProfileImage": "Upload profile image",
    "shared.users": "Users",
    "shared.valueOff": "{value} off",
    "shared.valuePerNight": "{value} / per night",
    "shared.viewCalendar": "View Calendar",
    "shared.weSentEmailToAddress": "We sent an email to {address}",
    "shared.weTextedYourSecurityCode": "We texted your security code to {phone}. It may take a minute to arrive.",
    "shared.weeklyDiscount": "Weekly Discount",
    "shared.weeklyPriceTooltip": "Weekly price with {percents} discount: {discount}",
    "shared.welcomeBack": "Welcome back!",
    "shared.whereAreYouGoing": "Where are you going?",
    "shared.withoutTitle": "Without title",
    "shared.yearNumber": "{number} Year",
    "shared.youAreReadyToPublish": "You're ready to publish!",
    "shared.youHaveSeenItAll": "Yay! You have seen it all",
    "shared.yourListingsCounter": "Your listings {count}",
    "shared.yourPasswordHasBeenUpdated": "Your password has been updated",
    "shared.zipCode": "Zip Code",
    "signIn.deactivatedByAdminModal.description": "Looks like you are trying to log in into an account which was previously deactivated by the administrator. Please, contact support for further details.",
    "signIn.deactivatedByUserModal.cancel": "No, keep it deactivated",
    "signIn.deactivatedByUserModal.confirm": "Yes, reactivate my account",
    "signIn.deactivatedByUserModal.description": "Looks like you are trying to log in into an account which was previously deactivated by you. You still can reactivate this account if you want it.",
    "signIn.description": "Enter your details to proceed further.",
    "signUp.dateOfBirth.info": "To sign up, you need to be at least 18. Your date of birth won't be shared with other people",
    "signUp.description": "Enter your details to proceed further.",
    "signUp.title": "Sign up to getting started",
    "sleepingArrangements.description": "Sharing the types of beds in each room can help guests understand the sleeping arrangements.",
    "sleepingArrangements.title": "Sleeping arrangements",
    "validations.avatarValidationError": "Please use JPG or PNG image smaller than 10MB",
    "validations.imageFormat": "File format must be JPG, JPEG, PNG",
    "validations.imageFormatAndMaxSize": "Image format: JPG or PNG. Max size: 10 Mb.",
    "validations.imageFormatMultiple": "Part of images canât be added: Some of them have not supported format. Please, upload JPG, JPEG, PNG",
    "validations.imageSize": "Image size must not exceed {value}MB",
    "validations.imageSizeMultiple": "Part of images canât be added: Some of them are larger than {value}Mb. Please, select smaller image",
    "validations.maxDigits": "Must be equal or less than {max} digits",
    "validations.maxPhotosDifference": "Maximum number of photos allowed to upload is {count}. You can upload up to {difference} more photos.",
    "validations.minAge": "You must be at least {years} years old to create an account",
    "validations.minBathrooms": "Please select at least {min} bathroom",
    "validations.minBeds": "Please select at least {min} Bed",
    "validations.minDigits": "Must be equal or more than {min} digits",
    "validations.minGuests": "Please select at least {min} guest",
    "validations.minRooms": "Please select at least {min} Room",
    "validations.numberOfBedsDoesNotMatch": "Number of beds does not match, please check it.",
    "validations.passwordsIdentical": "New password must not be identical with old",
    "validations.pleaseUploadTheImage": "Please upload the image",
    "validations.stringLength": "Must be equal {value} characters",
    "yup.mixed.required": "Can't be blank",
    "yup.number.max": "Must be equal or less than {max}",
    "yup.string.email": "Email has a wrong format",
    "yup.string.max": "Must be equal or less than {max} characters",
    "yup.string.min": "Must be at least {min} characters long",
  },
}
`;
